neqo-transport/tests/sim/taildrop.rs

```diff
@@ -151,6 +158,9 @@ impl TailDrop {
 impl Node for TailDrop {
     fn process(&mut self, d: Option<Datagram>, now: Instant) -> Output {
         if let Some(dgram) = d {
+            if self.start_time.is_none() {
+                self.start_time = Some(now);
+            }
```

Comment:
You should move this to the init() function, which exists for this purpose.  That won't capture when the first packet was injected into this link, but it is close enough (the server->client link will see a packet 1/2 RTT later in most cases).